src/libraries/Emitter.ts

```diff
@@ -102,32 +100,48 @@ export default class Emitter {
    * @example
    * me.listeners('ready');
    */
-  public listeners(event: string): (EventHandler | OnceEventHandler)[] {
+  public listeners(event: string): (EventHandler | OnceEventHandler)[] | undefined {
```

Comment:
no need to change anything here just want to share some thoughts 
 
Convention for return types it to use `void` over `undefined`. `type void = undefined | null` making it a little more flexible for what is returned. 

Although I have heard compelling arguments to never use `null` and only use `undefined` since then you only have one null like type